docs/proposals/remove_resolved_requirements.md

```diff
@@ -0,0 +1,51 @@
+# Removing requirements.txt
+- Authors: Ofek Lev
+- Date: 2018-12-20
+- Discussion: https://github.com/DataDog/integrations-core/pull/2800
+- Status: accepted
+
+## Origin
+
+At the start, there was only one dependency file: `requirements.txt`. It was used exclusively for our Ruby/rake tests. The contents appear to have come directly from the `dd-agent` and omnibus repos during the:
+
+Migration of checks to their own repo (integrations-core), e.g. https://github.com/DataDog/integrations-core/pull/50
+Porting of checks to wheels https://github.com/DataDog/integrations-core/pull/829
+
+## Port to wheels
+
+During the effort to make every check its own Python package, it was also decided that, to further the separation of the agents and their integrations, the builders should install each check as a wheel, thus setting up the scaffolding to move to using every check’s defined dependency file rather than a single large file of dependencies. See:
+
+https://github.com/DataDog/dd-agent-omnibus/pull/213
+https://github.com/DataDog/datadog-agent/pull/1048
+
+## New testing framework
+
+During review of the PoC for tox/pytest, security asked us to define what is required and then resolve the dependencies in a separate file to ease use of security scanners and improve security when installing the wheels. The pinned file became `requirements.in` and the resolved file became `requirements.txt`. See https://github.com/DataDog/integrations-core/pull/1024#discussion_r162973079
+
+The work to unpack each wheel to verify dependency hashes had not been done yet (and never was) so we began by using `requirements.in` in every `setup.py`.
```

Comment:
I think you should remove this sentence, it makes the document less clear. The work was done it was just incomplete (and you detail why it was incomplete below)

Reply:
You don't actually say this is why we're not using it though.

Reply:
resolved offline